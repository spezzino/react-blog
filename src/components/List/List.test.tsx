import React from "react";
import { List } from "./";
import renderer from "react-test-renderer";

const SampleComponent = (props: any) => {
  return <div>{props.id}</div>;
};

test("List gets populated with items", () => {
  const component = renderer.create(
    <List items={[{ id: 1 }]} component={SampleComponent} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree?.children?.length).toEqual(1);
});
