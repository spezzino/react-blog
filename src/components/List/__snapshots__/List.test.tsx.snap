// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List gets populated with items 1`] = `
<div>
  <div>
    1
  </div>
</div>
`;
